src/osd/scheduler/mClockScheduler.h

```diff
@@ -130,7 +131,10 @@ class mClockScheduler : public OpScheduler, md_config_obs_t {
     true,
     true,
     2>;
+  typedef std::map<unsigned, std::list<OpSchedulerItem>, std::greater<unsigned>> SubQueue;
+  typedef SubQueue::iterator Iter;
```

Comment:
That's a too general name to create. Possible options:
1) 'using SubQueueIter = SubQueue::iterator'
2) not defining a special name for it, as on most instances you'd use 'auto'
(I'll continue reading, to see what makes sense)

Reply:
To be fair, we have something of a historical precedent for using very generic names in these kinds of situations. ;)  But I do think something like SubQueueIter would be a bit more descriptive.